src/components/TileVideo/index.js

```diff
@@ -6,19 +6,18 @@ import VideoPlayer from '../VideoPlayer'
 import Unmute from '../Icons/Muted'
 import Mute from '../Icons/Unmuted'
 
-const ACCOUNT_ID = '5344802162001'
```

Comment:
Safe to remove because of the previous video player changes, right?

Reply:
safe to remove because the video player already sets these values by default